lib/Http/RequestHelper.php

```diff
@@ -82,17 +82,32 @@ public function getRequest(Request $request = null)
      */
     public function hasMasterRequest(): bool
```

Comment:
I'd say this should be deprecated and replaced by `hasMainRequest()`, too.

Reply:
Absolutely 😊 See b3fd6ee1370c764478daaf1a50fe66632ec6067d